### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ineiti**
Clean up the README.md in the top-directory and add READMEs in the app-directories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,18 @@
...
+
+It works closely together with the cryptographic-library found in `dedis/crypto`
+and allows for setting up of *protocols*, *services*, and "apps". A protocol will
+send back and forth messages, mostly in a tree-based structure of
+nodes, but it can also broadcast or bypass the tree.
+A service interacts with clients and will spawn and wait for the result
+of different protocols.
+An app is an example of a user-space program that can communicate to one or more
+services of a cothority.
+You can find a list of protocols and services supported later.
```
Comment (nikkolasg) [inline README.md]: later => where ? end of document ?

File: README.md [hunk 2]
```diff
@@ -59,223 +47,44 @@ https://github.com/dedis/cothority/tree/v0
...
-with
-
-```bash
-cosi verify msg "Hello CoSi" -sig msg.sig
-```
-
-If you would instead like to sign a message contained in a file you 
-specify (which may be either text or arbitrary binary data), you can do 
-this as follows:
+There are three apps available:
```
Comment (Liamsi) [inline README.md]: Why is there another section with "Apps" then with more than three?

File: README.md [hunk 3]
```diff
@@ -3,30 +3,17 @@
...
-* Enhancing Bitcoin Security and Performance with Strong Consistency via Collective Signing: [paper](http://arxiv.org/abs/1602.06997)
- 
+It offers a framework for simulating and deploying decentralized and 
+distributed cryptographic protocols.
+
+It works closely together with the cryptographic-library found in `dedis/crypto`
+and allows for setting up of *protocols*, *services*, and "apps". A protocol will
+send back and forth messages, mostly in a tree-based structure of
+nodes, but it can also broadcast or bypass the tree.
+A service interacts with clients and will spawn and wait for the result
```
Comment (nikkolasg) [inline README.md]: waits ?

File: README.md [hunk 4]
```diff
@@ -332,11 +150,11 @@ hosts which form a tree. Every protocol defines the steps needed to
...
-
-* [`lib/`](lib/): contains all internally used libraries
-* [`lib/sda/`](lib/sda/): basic definition of our framework
-* [`protocols/`](protocols/): one directory per protocol, holds both the 
-definition and eventual initialisation necessary for a simulation
-* [`simul/`](simul/): used for running simulations on localhost and 
-DeterLab
+## Directory-structure
+
+* [`sda/`](sda/): basic definition of our framework
```
Comment (nikkolasg) [inline README.md]: main framework which is used to build the protocols: it handles the network layer, the overlay tree layer & the services.
something more descriptive ?

File: app/status/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Installation
```
Comment (nikkolasg) [inline app/status/README.md]: Maybe again a very quick one-sentence explanation of what is the status app ?

File: README.md (hunk 3 above)
Comment (ineiti) [inline README.md]: nice try - not here ;) It's "will span and [will] wait"

File: README.md (hunk 4 above)
Comment (ineiti) [inline README.md]: the framework which is used to handle the messages, the overlay tree layer, the protocols and the services
??

**Comment by nikkolasg** [comment]
After the small one sentence explanation for status, I'm OK

**Comment by nikkolasg** [comment]
I'm merging as I am the only one left at work these next days... :D

**Pull request merged**